README.md

```diff
@@ -1,15 +1,15 @@
 # Recursion Problems
 
 ## Definitions
-Define the following: 
+Define the following:
 
-- Recursion
-- Recursive Case
-- Base Case
-- Activation Chain/Stack
-- Activation Record/Call
-- Infinite Recursion/Stack Overflow/Stack too deep
-- Tail Recursion
+- Recursion : recurrence or repetition,
+- Recursive Case: repeated application of a recursive statement
+- Base Case: the part of a recursive definition or algorithm that is not defined in terms of itself
+- Activation Chain/Stack: the loops of the recursion being executed
+- Activation Record/Call: the data structure that composes a call stack
```

Comment:
This is not correct.

An activation call is a single call to the recursion method. One of the "bubbles"